utils/cargo-gbuild/program/Cargo.lock

```diff
@@ -0,0 +1,643 @@
+# This file is automatically @generated by Cargo.
```

Comment:
`gear_wasm_builder/test_program` has `build script` which includes the instructions of `gear-wasm-builder`, stands for different purpose

Reply:
whats the differ?

Reply:
if testing with `gear_wasm_builder/test_program`, the test program will be built for 3 times, 

1. (native build) cargo build
2. (build script) gear-wasm-builder with shadow manifest
3. (gbuild) wasm build

which is not expected, the artifacts could be messed up since build script generates the wasm artifacts as well and we will not be able to detect if the `build` process works fine in `gbuild`